helpers/jwt_service.go

```diff
@@ -39,3 +38,24 @@ func GenerateToken(userId string) (string, error) {
 	token, err := claims.SignedString([]byte(jwtServiceCreds.secretKey))
 	return token, err
 }
+
+/*
+This function validates a given token string, and if successful, returns a string representation of the ObjectID
+of the user who created the token. If unsuccessful, the string is empty, and the error is returned instead.
+*/
+func ValidateToken(tokenString string) (string, error) {
+	// Convert the token string to a JWT token, and verify that it was created with our secret key.
+	token, err := jwt.ParseWithClaims(tokenString, &jwt.StandardClaims{}, (func(token *jwt.Token) (interface{}, error) {
+		return []byte(jwtServiceCreds.secretKey), nil
+	}))
+	if err != nil {
+		return "", err
+	}
+	// Type assertion so that we can extract specific fields from our claims object.
+	claims, ok := token.Claims.(*jwt.StandardClaims)
+	if ok && token.Valid {
+		return claims.Subject, nil
+	}
+	return "", err
+
```

Comment:
nit: excess newline

Reply:
also maybe return nill instead of "", although it doesn't matter too much.

Reply:
Also you have `return "", err` twice. You can potentially avoid violating DRY by refactoring your code to something like
```go
if err == nil {
  // Type assertion so that we can extract specific fields from our claims object.
  claims, ok := token.Claims.(*jwt.StandardClaims)
  if ok && token.Valid {
    return claims.Subject, nil
  }
}
else {
  return nil, err
}
```

Reply:
Excess newline is from `make clean`. Changed "" to nil just to be consistent across the board. Refactored code.